setup.py

```diff
@@ -18,24 +18,24 @@
     "Operating System :: OS Independent",
 ]
 DEPENDENCIES = [
-    "pandas>=1.0.0",
-    "numpy>=1.21.0, <2.0.0",
-    "scipy<=1.13.1",
-    "matplotlib>=3.8.0",
+    "numpy>=2.0.0",
+    "pandas>=2.2.2",
+    "scipy>=1.14.0",
+    "xarray>=2024.6.0",
+    "matplotlib>=3.9.1",
+    "scikit-learn>=1.5.1",
+    "h5py>=3.6.0",
+    "h5pyd >=0.7.0",
+    "netCDF4>=1.7.1.post1",
+    "statsmodels>=0.14.2",
     "requests",
     "pecos>=0.3.0",
     "fatpack",
     "lxml",
-    "scikit-learn",
     "NREL-rex>=0.2.63",
     "six>=1.13.0",
-    "h5py>=3.6.0",
-    "h5pyd >=0.7.0",
-    "netCDF4>=1.5.8, <=1.6.5",
-    "xarray",
-    "statsmodels",
     "pytz",
-    "bottleneck",
+    "bottleneck>=1.4.0",
     "beautifulsoup4",
 ]
```

Comment:
```suggestion
    "notebook",
]

Reply:
`notebook` is not a requirement of MHKiT. It could reasonably be added so we know users have it when they go to run the examples. Is that your goal here?

Reply:
Yes, this was included in a previous update to the requirements.txt and environment.yml, but must have been missed here in setup.py. I'm open to leaving this for future discussion if you think it may pose an issue.